web/src/lib/components/shared-components/upload-panel.svelte

```diff
@@ -34,13 +34,12 @@
     in:fade={{ duration: 250 }}
     out:fade={{ duration: 250 }}
     on:outroend={() => {
-      notificationController.show({
-        message:
-          ($errorCounter > 0
-            ? `Upload completed with ${$errorCounter} error${$errorCounter > 1 ? 's' : ''}`
-            : 'Upload success') + ', refresh the page to see new upload assets.',
-        type: $errorCounter > 0 ? NotificationType.Warning : NotificationType.Info,
-      });
+      if ($successCounter > 0 && $successCounter !== $duplicateCounter) {
```

Comment:
You can push more commits to a PR, you don't have to create a new one. The root cause appears to be incrementing `$successCounter` for duplicate assets, we should fix that instead of creating a workaround. Also the error count is no longer reported, is that an intentional change?